src/Forms/Forms.Component.js

```diff
@@ -55,6 +55,48 @@ export const FormsComponent = () => {
                 </div>
             </Example>
 
+            <Example
+                title='Inline Labels'>
+                <div>
+                    <FormGroup>
+                        <FormItem isHorizontal>
+                            <FormLabel htmlFor='input-1'>Default Input</FormLabel>
+                            <FormInput id='input-1' placeholder='Field placeholder text'
+                                type='text' />
+                        </FormItem>
+                    </FormGroup>
+
+                    <FormGroup>
+                        <FormItem isHorizontal>
+                            <FormLabel htmlFor='input-2' required>
+                                Required Input
+                            </FormLabel>
+                            <FormInput id='input-2' placeholder='Field placeholder text'
+                                type='text' />
+                        </FormItem>
+                    </FormGroup>
+
+                    <FormGroup>
+                        <FormItem isHorizontal>
+                            <FormLabel htmlFor='input-3' required>
+                                Password
```

Comment:
> When a field is required, the label should include an asterisk (*).

Does that mean the user is responsible for adding an `*` and this should be `Password *` ?

Reply:
Nah, our docs need an entire story just to rewrite. I'll fix this now though.